shoop/utils/multilanguage_model_form.py

```diff
@@ -87,6 +87,12 @@ def __init__(self, **kwargs):
         kwargs["initial"] = object_data
         super(MultiLanguageModelForm, self).__init__(**kwargs)
 
+        if all([self.instance, self.instance.pk, hasattr(self.instance, "protected_fields")]):
```

Comment:
You're using this same idiom (`if ...: for field in ...:`) in both these functions; might be worth to refactor it to

```python
def _get_protected_fields(self):
    if self.instance and self.instance.pk:
        return getattr(self.instance, "protected_fields", ())
    return ()
```

(with a suitable docstring, of course ;) )